flytekit/taskplugins/pod/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .task import Pod
+
+__all__ = [Pod]
```

Comment:
so the `__all__`` should be a list of string names of the modules you want to expose. 
I think you can just skip it, refer to my new PR